docs/how-to/default-profile.md

```diff
@@ -0,0 +1,96 @@
+
+If previously you configured your IPFS node to use another profile, let's say `badgerds`, the above command would give you an output similar to the following:
+
+```bash
+"prefix": "badger.datastore",
+```
+
+This output line refers to `badgerds` profile .
+
+## Available profiles
+
+Here's a list of all the profiles available for your IPFS node. You can compare the output from above command with any of the following profiles to find what profile is currently being used by your node:
+
+- `flatfs`
+- `badgerds`
+- `server`
+- `randomports`
+- `default-datastore`
+- `local-discovery`
+- `test`
+- `default-networking`
+- `lowpower`
+
+## Reset your profile
+
+If you want to reset your IPFS node profile to the default profile, you can do so by using the following method:
+
+```diff
+ipfs config profile apply default-datastore
+
+...
+"Datastore": {
+    "BloomFilterSize": 0,
+    "GCPeriod": "1h",
+    "HashOnRead": false,
+    "Spec": {
+-       >> "child": {"path":"badgerds","syncWrites":false,"truncate":true,"type":"badgerds"},
++       << "mounts": [{"child":{"path":"blocks","shardFunc":"/repo/flatfs/shard/v1/next-to-last/2","sync":true,"type":"flatfs"},"mountpoint":"/blocks","prefix":"flatfs.datastore","type":"measure"},{"child":{"compression":"none","path":"datastore","type":"levelds"},"mountpoint":"/","prefix":"leveldb.datastore","type":"measure"}],
+-       >> "prefix": "badger.datastore",
+        <> "type": "measure"
+        ** "type": "mount"
+    },
+    "StorageGCWatermark": 90,
+    "StorageMax": "10GB"
+},
+...
+```
+
+Above command shows the difference between your existing IPFS config and the new configuration. Also, it would overwrite your IPFS `config` file, so it's always better to create a backup of your existing IPFS configuration before running this command.
+
+## Converting profiles
+
+Not all profiles are compatible with each other simply because they use very different technologies for storing the data inside the datastores. For instance, if you want to convert `badgerds` to `default-datastore`, you would have to use another helper tool called [ipfs-ds-convert](https://dist.ipfs.io/#ipfs-ds-convert) to convert the datastore to the required format.
+
+Install `ipfs-ds-convert` tool from the given link and run the following command to finally convert the datastore:
+
+```bash
+ipfs-ds-convert convert
+
+> convert 2020/12/06 21:27:26 Checks OK
+> convert 2020/12/06 21:27:27 Copying keys, this can take a long time
+> copied 2002 keys
+> convert 2020/12/06 21:29:01 All data copied, swapping repo
+> convert 2020/12/06 21:29:02 Verifying key integrity
+> convert 2020/12/06 21:29:02 2002 keys OK
+> convert 2020/12/06 21:29:02 Saving new spec
+> convert 2020/12/06 21:29:02 All tasks finished
+```
+
+This can take a very long time to complete depending on the size of the datastore. If running this on a headless server it's recommended to use something like `screen` or `tmux` to run this command in a persistent shell.
```

Comment:
```suggestion
This can take a very long time to complete. If you are running this on a headless server we recommend you use something like `screen` or `tmux` to run this command in a persistent shell.
```